### DISCUSSION TITLE
how to add a custom action to slash plugin which adds arbitrary markdown to the editor state?

### DISCUSSION THREAD

**Discussion by G3root** [Milkdown/milkdown#201] [Q&A]
is there any way to insert arbitrary  markdown like below by an action

  ```` 
# Project Title
## Run Locally
### Run Locally
````

**Comment by Saul-Mirone** [comment]
There's no direct API to do this, but it won't hard, you can do something like this:
https://codesandbox.io/s/milkdown-listener-count-eteqe?file=/src/index.ts:1427-1572
